functors_task_1/student/include/impl.h

```diff
@@ -16,16 +24,20 @@ struct BeerOrganizer
  *
  * @note Only Corona and HoeGaarden are expensive
  */
-bool isExpensiveBeer(/**???*/)
+bool isExpensiveBeer(BeerBrand beer)
 {
+    if(beer == BeerBrand::Corona || beer == BeerBrand::HoeGaarden) return true;
```

Comment:
please simplify, try to avoid naked true and false in your code
you may directly return result of the expression